packages/eslint-plugin-lit-a11y/template-analyzer/util.js

```diff
@@ -138,6 +138,19 @@ function getExpressionPlaceholder(node, quasi) {
 
 exports.getExpressionPlaceholder = getExpressionPlaceholder;
 
+/**
+ * Checks to see if an element is the root HTML element
+ *
+ * @param {import("estree").TaggedTemplateExpression} node Root node
+ *
+ * @return {boolean}
+ */
+function isRootElement(node) {
+  return /<html([\s\S]*)<\/html>/.test(node);
```

Comment:
would you like me to annotate the code with what the regex is doing? i'd be more than happy to!

Reply:
That would be great!

Reply:
Done! I added a link to the [resource](https://regex101.com/r/lKJKFv/1/) which gives a breakdown of what each step of the regex does.